tuxemon/core/main.py

```diff
@@ -39,8 +39,7 @@
 
 logger = logging.getLogger(__name__)
 
-
-def main(load_slot=None):
+def main(arguments=[]):
```

Comment:
please revert this change.  we should not be using arguments from the CLI internally.